packages/SwingSet/src/lib/netstring.js

```diff
@@ -54,6 +54,12 @@ export function decode(data) {
       // reject NaN, all negative numbers
       assert.fail(X`unparseable size ${sizeString}, should be integer`);
     }
+    if (optMaxChunkSize) {
+      assert(
+        size < optMaxChunkSize,
```

Comment:
nit: `<=` rather than `<`, if the option is really the "maximum size"

not a big deal but if someone builds an alternate implementation and follows the variable name rather than the implementation, they could wind up off-by-one